### DISCUSSION TITLE
Saleor Flat Tax Plugin

### DISCUSSION THREAD

**Discussion by carlosa54** [saleor/saleor#8341] [Show and tell]
Hello, I recently published a plugin I made to calculate custom flat taxes without connecting to external services. My use case required handling a small amount of taxes for only one country. So Avalara/Vatlayer was an overkill for me.

The plugin is based out of the Vatlayer plugin but modified to use the taxes configured through the plugin configuration. I have been testing it on my Saleor instance and everything works as expected.

Here's the repo if anyone is interested: https://github.com/carlosa54/saleor-flat-tax-plugin

**Comment by carlosa54** [comment]
Here how it looks:

<img width="964" alt="plugin-config" src="https://user-images.githubusercontent.com/4250593/136877078-c03d5baf-193a-4843-ae9f-cb046f67dc98.png">


<img width="658" alt="product-type-taxes" src="https://user-images.githubusercontent.com/4250593/136877122-a945f624-7ba2-42ce-af2a-c419c2f40ec7.png">
<img width="304" alt="product-taxes" src="https://user-images.githubusercontent.com/4250593/136877131-76f3a61e-b1db-46ca-8e9a-24316d8529df.png">

**Reply by tmechsner** [reply]
Thanks a lot! Same use-case here. Much appreciated! :)
